tests/unit/many-array-test.js

```diff
@@ -0,0 +1,233 @@
+import { moduleForModel, test } from 'ember-qunit';
```

Comment:
This seems kind of like an acceptance test. Maybe it should move there, I'm not sure.

Reply:
agreed, moved